style.css

```diff
@@ -1 +1,727 @@
-/*  Add your styling here */
+
+  
+ /*
+ MOBILE
+
+ .homepage-container {
+    display: grid;
+    grid-template-areas:
+    "hero       hero        hero        hero"
+    "content    content     content     content"
+    "cake      cake         cake        cake"
+    "flex       flex        flex        flex"  
+    "footer     footer      footer      footer";
+}
+*/
+
+/*   
+TABLET
+
+        display: grid;
+        grid-template-areas:
+        "hero       hero        hero        hero"
+        "menu       menu        menu        menu"
+        "cake       cake        content     content"
+        "flex       flex        flex        flex" 
+        "footer     footer      footer      footer";
+       }
+*/
+
+/*   
+DESKTOP
+
+.homepage-container {
+      display: grid;
+      grid-template-areas:
+        ".  hero    hero      ."
+        ".  menu    menu      ."
+        ".  cake    content   ."
+        ".  flex    flex      ." 
+        ".  footer  footer    .";
+*/
+
+/*
+IMPORTED FONTS FROM
+    https://fonts.google.com/specimen/Fuzzy+Bubbles#standard-styles
+*/
+
+@font-face {
+    font-family: FuzzyBubbles;
+    src: url("img/FuzzyBubbles-Regular.ttf") 
+    }
+
+@font-face {
+        font-family: myFirstFont;
+        src: url("img/FuzzyBubbles-Bold.ttf");
+        font-weight: bold;
+}
+      
+
+.hero-box {
+    width: 100%;
+    background-color: #fce0df;    
+    grid-area: hero;
+ }
+  
+.hero-flexbox
+  {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+}
+
+/* 
+The Logo for Asterisk Et Obelix Cakes
+*/
+  
+.AsterixetObelix {
+  width:9.375em; /* 150px; THUMBNAIL SIZE */
+  object-fit: contain;
+  padding-top:1em;
+}
+
+.menu-header {
+  display:none;
+}
+
+.tablet-prompt{
+  display:none;
+}
+
+.desktop-prompt{
+  display:none;
+}
+
+
+/* Two ways of displaying the heading depending on the screen size */
+/* If screen size is 480px wide, or less: */
+@media screen and (max-width:480px) {
+
+
+      .mainheading {
+          font-size:140%;
+          line-height: 1.3em;
+          text-align: center;
+          padding-right:1.2em;
+      }
+
+      .thequote {
+        padding-left:1.7em;
+        text-align: justify;
+      } 
+
+  
+}
+
+/* Otherwise: */
+@media screen and (min-width: 481px) {
+
+      .mainheading{
+          font-size:400%;
+          text-align:right;
+          margin-left:12em;
```

Comment:
This huge margin is breaking your layout - remove it.
You shouldn't need it - something like setting the header to flex and using `space-between` should naturally keep the logo to the left and the heading to the right.